agendas/2020-11-25.md

```diff
@@ -2,17 +2,17 @@
 
 The GraphQL Working Group meets monthly to discuss proposed changes to the [GraphQL-JS](https://github.com/graphql/graphql-spec) library, other related foundation libraries like [express-graphql](https://github.com/graphql/express-graphql) and [graphql-relay-js](https://github.com/graphql/graphql-relay-js) and other
 relevant topics to core Javascript GraphQL projects. This is an open meeting in which
-anyone in the GraphQL community may attend. *To attend this meeting or propose
-agenda, edit this file.*
+anyone in the GraphQL community may attend. _To attend this meeting or propose
+agenda, edit this file._
```

Comment:
@gagoar Are these changes caused by prettier?

Reply:
these are changes produces by vscode markdown editor. happy to roll this back. looks like is a good practice to use _ instead of *